WebApp/handler/TransformationMonitorHandler.py

```diff
@@ -273,14 +276,20 @@ def __fileRetry(self, prodid, mode):
   def __dataQuery(self, prodid):
     callback = {}
     tsClient = TransformationClient()
-    res = tsClient.getTransformationInputDataQuery(prodid)
+
+    # FIXME: getTransformationInputDataQuery has been replaced by getTransformationMetaQuery in DIRAC v7r0
+    try:
```

Comment:
There won't be an exception, the Client will just forward any call to the service handler. One has to check the res['Message'] for the Unknown method string

Reply:
Of course! Fixed, thanks.